main.go

```diff
@@ -134,3 +132,84 @@ func isValidEvent(event fsnotify.Event) bool {
 	//}
 	return true
 }
+
+func startFluentbit() {
```

Comment:
I think we can make the function name short:
startFluentbit => start
waitFluentbit => wait
delay => backoff
KillFluentbit => stop